CONTRIBUTING.md

```diff
@@ -51,6 +51,44 @@ Work on your self-assigned issue and eventually create a Pull Request.
    pre-commit install
    ```
 
+1. On MacOS with Apple Silicon, if this installation approach runs into errors, you may need to install the following prerequisites:
```

Comment:
all the lines have this 1. preceding them, not sure what the goal is, in theory in markdown that's an ordered list i beleive?

Reply:
@w4nderlust -- indeed I made a typo -- will fix as part of the next commit.  Thank you.

Reply:
@w4nderlust Update -- I checked; not a typo -- continuing the ordered was exactly my intention.  I will check to make sure that things render correctly.  Thanks.